### ISSUE TITLE
View latest post doesnt work exactly as intended

### ISSUE THREAD

**Issue by RBRat3** [ppy/osu-web#5024]
Im under the assumption that viewing latest post in a topic list puts you in the last post you read as a sort of post resume. As someone who doesn't regularly read the forums there could be 100s of pages between my last read state where I have to manually input the last page button in the bottom to actually get the "latest post" to see the current topic at hand. May I suggest adding a last read button and an actual latest post button or something to that effect?

In the video link below you can see as I click on latest post and go back and forth pages are added up as they're being counted as read but never goes to latest post.
https://puu.sh/ElMEc/95d8f3501c.mp4
Firefox 69.0.1 (64-bit)

**Comment by peppy** [comment]
I also find this behaviour a bit hard to follow. In most cases I actually want to go to the newest post, not last unread.

Interested to get more opinions on this one.

**Comment by peppy** [comment]
@arflyte interested in your thoughts on this one.

**Comment by arflyte** [comment]
Yeah, I have to agree that's a bit weird. I think it makes more sense to go to the first post of the latest thread, with the button says "View Latest Thread".
